### PR TITLE
update documentation

### PR DESCRIPTION
**Pull request by rchamarthy**
Signed-off-by: Ravi Chamarthy <ravchama@cisco.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
-# calc
+# calc - calculator implementation in python
 
-calculator implementation in python
+Calculator provides basic math functions like:
+
+    * add
```
Comment (Ace369) [inline README.md]: Describe what add does.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+    * subtract
```
Comment (IJ0717) [inline README.md]: describe subtract

File: README.md [hunk 3]
```diff
@@ -1,3 +1,12 @@
...
+    * multiply
```
Comment (suhanimv) [inline README.md]: obtain a number from multiplying two different numbers. ex. 6*8=48

File: README.md [hunk 4]
```diff
@@ -1,3 +1,12 @@
-# calc
+# calc - calculator implementation in python
 
-calculator implementation in python
```
Comment (Ace369) [inline README.md]: Capitalize the titles.

**Review by suhanimv — CHANGES_REQUESTED** [review]

**Review by suhanimv — APPROVED** [review]

**Review by IJ0717 — APPROVED** [review]

**Review by Ace369 — APPROVED** [review]

**Pull request merged**